script/pgsqlms

```diff
@@ -1021,6 +1036,11 @@ sub pgsql_validate_all {
     @content = <$fh>;
     close $fh;
 
+    unless ( $maxlag =~ /^[0-9,.E]+$/ ) {
+	ocf_exit_reason( 'maxlag is not a number: "%s"', $maxlag );
+        exit $OCF_ERR_ARGS;
```

Comment:
This should probably returns `$OCF_ERR_INSTALLED` as this is an error related to the setup of the whole resource, not only the local PostgreSQL instance.